platform/src/apis/Platform.Api.Benchmark/Comparators/ComparatorsFunctions.cs

```diff
@@ -1,86 +1,23 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Net;
-using System.Threading.Tasks;
+﻿using System.Net;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Azure.Functions.Worker.Http;
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
-using Microsoft.Extensions.Logging;
-using Platform.Functions.Extensions;
 using Platform.Functions.OpenApi;
 namespace Platform.Api.Benchmark.Comparators;
 
-public class ComparatorsFunctions(ILogger<ComparatorsFunctions> logger, IComparatorSchoolsService schoolsService, IComparatorTrustsService trustsService)
+public class ComparatorsFunctions
 {
     [Function(nameof(SchoolComparatorsAsync))]
-    [OpenApiOperation(nameof(SchoolComparatorsAsync), "Comparators")]
+    [OpenApiOperation(nameof(SchoolComparatorsAsync), "Comparators", Deprecated = true)]
```

Comment:
Nice

Reply:
Just trying to be kind! I guess at some point when the APIs are opened up we're going to want to think about versioning...